lib/helpers/site.ts

```diff
@@ -0,0 +1,7 @@
+export const getSiteFromLocale = (locale: string) => {
```

Comment:
extracted these from the big utils file which mixed client and server code, separation of the two is critical for app directory